src/keycloak/keycloak-ees-realm.json

```diff
@@ -448,7 +448,7 @@
     "emailVerified" : true,
     "firstName" : "Bau1",
     "lastName" : "User1",
-    "email" : "ees-bau1@education.gov.uk",
+    "email" : "ees-test.bau1@education.gov.uk",
```

Comment:
Just out of interest, I think this config here is setting up these users? If so, I assume that `secretData` is basically where the password is set?

I ask because I can see a potential pain point of trying to get passwords aligned between AAD and KeyCloak - maybe if someone does change the UI tests so they work with KeyCloak, might be worth to create separate environment variables to store the passwords used by KeyCloak, so people don#'t need to change their `.env` files if they're switching between AAD and KeyCloak. Maybe another solution is they could also specify a separate `.env` file for KeyCloak

Reply:
Not too sure on this but I'm planning to load `.env.keycloak`  in the UI tests if a user wants to use keycloak as an identity provider